src/site/content/en/blog/url/index.md

```diff
@@ -0,0 +1,290 @@
+    two-part top-level domain ('public suffix') and the third-level domain name just before that:
+    for example, `example.co.uk` or `example.com.au`.
+
+{% Aside %}
+Confusingly, perhaps, domains such as `example.github.io` or `example.glitch.me`
+are  [defined as 'registrable domains'](https://url.spec.whatwg.org/#host-registrable-domain)
+though they cannot be registered via a domain name registrar.
+{% endAside %}
+
+### Scheme {: #scheme}
+
+The part of the URL (before `://`) that defines the [protocol](https://developer.mozilla.org/docs/Glossary/Protocol)
+(or action to be taken by the user agent) when a request is made to a URL. For example, a request
+to a URL with an `https` scheme should be made using the [HTTPS protocol](https://en.wikipedia.org/wiki/HTTPS).
+For a request to a URL with a scheme such as `file`, `mailto` or `git` that doesn't correspond to a
+network protocol, the behaviour depends on the user agent. For example, when a user clicks on a
+`mailto` link, the browser might open its default email application, using the values in the link
+URL.
+
+[Spec](https://url.spec.whatwg.org/#dom-url-protocol) • [API](https://developer.mozilla.org/docs/Web/API/URL/protocol)
+
+### Search (or 'query string') {: #search}
+
+A series of key/value pairs that represent [parameters](#parameter), at the end of a URL after a
+question mark.
+
+-  For `https://example.com/cats?pattern=tabby&mood=bonkers`, the search string is
+    `pattern=tabby&mood=bonkers`.
+
+[Spec](https://url.spec.whatwg.org/#dom-url-search) • [API](https://developer.mozilla.org/docs/Web/API/URL/search)
+
+### Second-level domain {: #sld}
+
+The domain before the [top-level domain](#tld):
+
+-  For the URL `https://www.example.com`, `example.com` is the second-level domain, a
+    [subdomain](#subdomain) of the top-level domain `com`.
+-  For `https://example.com.au`, the top-level domain is `au`, the second-level domain is `com`
+    and the third-level domain is `example`. In this example, `com.au` is a subdomain of `au` and
+    `example.com.au` is a subdomain of `com.au`.
+
+### Site {: #site}
+
+{% Aside %}
+According to the [URL Standard](https://url.spec.whatwg.org/), site includes [scheme](#scheme).
+(See [discussion of 'same site'](https://github.com/whatwg/url/issues/448).)
+{% endAside %}
+
+-   For a site such as `https://example.com` with a single-part top-level domain: the scheme, plus
+    the top-level domain and the part before that. For example, for the URL
+    `https://www.example.com/foo`, the site is `https://example.com`.
+-   For an [eTLD](#tld) where only third-level registrations are allowed (such as `co.uk` or
+    `github.io`): the scheme, the top-level domain ('public suffix') and the part of the domain name
+    just before that. For example, for the URL `https://www.example.co.uk/foo`, the site is
+    `example.co.uk`. (Confusingly, perhaps, domains such as `example.github.io` or `example.glitch.me`
```

Comment:
Should this have the scheme, as per above explanation?

```suggestion
    `https://example.co.uk`. (Confusingly, perhaps, domains such as `example.github.io` or `example.glitch.me`
```

Reply:
Oops — yes! Thank you.